src/sql/workbench/contrib/query/browser/queryResultsEditor.ts

```diff
@@ -76,8 +76,8 @@ export class QueryResultsEditor extends EditorPane {
 	public static ID: string = 'workbench.editor.queryResultsEditor';
 	protected _rawOptions: BareResultsGridInfo;
 
-	private resultsView: QueryResultsView;
-	private styleSheet = DOM.createStyleSheet();
+	private resultsView?: QueryResultsView;
+	private styleSheet?= DOM.createStyleSheet();
```

Comment:
This shouldn't be potentially undefined, DOM.createStyleSheet returns a style sheet and the only place we're removing it is in dispose(), where you can just put a ! at the end of it. Would make a lot of the code simpler too.

Reply:
Glad to see we had the same thoughts here for a lot of this stuff 😆